src/pages/jobs/[id].astro

```diff
@@ -98,26 +82,16 @@ const freelancerImage: ImageMetadata | undefined = imageFiles[imagePath];
               <span class="text-gray-500 dark:text-gray-400">Image not available</span>
             </div>
           )}
-          <div class="flex space-x-4 m-4 text-center">
-            {freelancer.forumProfile && (
-              <a href={freelancer.forumProfile} target="_blank" rel="noopener noreferrer" class="text-neutral-800 dark:text-neutral-100">
-                <Image 
-                  src={forumIcon}
-                  alt="CommerceQuest Forum Profile"
-                  width={32}
-                  height={32}
-                  class="w-8 h-8"
-                />
-              </a>
-            )}
+          <div class="flex items-center mb-4 mt-4 ml-3 gap-2">
+            <CQIcon profileUrl={freelancer.forumProfile} size="sm" />
```

Comment:
forumProfile is an optional field; we should have the check here again as you had before

Reply:
Done!